lib/money-rails/active_record/monetizable.rb

```diff
@@ -85,10 +85,10 @@ def monetize(*fields)
             #
             # To disable validation entirely, use :disable_validation, E.g:
             #   monetize :price_in_a_range_cents, :disable_validation => true
-            if validation_enabled = MoneyRails.include_validations && !options[:disable_validation]
+            if (validation_enabled = MoneyRails.include_validations && !options[:disable_validation])
```

Comment:
"Note that there is an exception to this rule, namely safe assignment in condition." https://github.com/bbatsov/ruby-style-guide#safe-assignment-in-condition